assets/js/base/components/cart-checkout/shipping-rates-control/index.js

```diff
@@ -11,6 +11,10 @@ import LoadingMask from '@woocommerce/base-components/loading-mask';
  * Internal dependencies
  */
 import Packages from './packages';
+import {
+	getShippingRatesPackageCount,
```

Comment:
I don’t know, I feel the variable name, and the extra hassle of importing it and assigning it and remembering what it does and where is much larger than the actual abstraction by itself, what do you think?

Reply:
Yeah, I think it's a way to abstract the fact that the variable is named `shippingRates` but in fact it's an array of packages. I don't have any strong opinion on this but would like to hear @mikejolley's thoughts.